packages/@o3r/rules-engine/schematics/rules-engine-to-component/index.spec.ts

```diff
@@ -53,6 +53,41 @@ describe('Enable rules-engine on component', () => {
     expect(fileContent).toMatch(/ngOnDestroy[^{]*{[^}]*this.rulesEngineService.disableRuleSetFor\(this.componentName\);/m);
   });
 
+  it('should update the module file', async () => {
+    initialTree.overwrite('test-folder/test-component.ts', `
+      import {Component} from '@angular/core';
+      import {O3rComponent} from '@o3r/core';
+      @O3rComponent({
+        componentType: 'Component'
+      })
+      @Component({
+        selector: 'empty-component',
+        template: '',
+        standalone: false
+      })
+      export class EmptyComponent {}
+    `);
+    initialTree.create('test-folder/test-component-module.ts', `
+      import { NgModule } from '@angular/core';
+      import { EmptyComponent } from './test-component';
+
+      @NgModule({
+        imports: [],
+        declarations: [EmptyComponent],
+        exports: [EmptyComponent]
+      })
+      export class EmptyModule {}
+    `);
+    const runner = new SchematicTestRunner('schematics', collectionPath);
+    const tree = await runner.runSchematic('rules-engine-to-component', {
+      projectName: 'test-project',
+      path: 'test-folder/test-component.ts'
+    }, initialTree);
+
+    const fileContent = tree.readContent(tree.files.find((file) => /test-component-module\.ts/.test(file)));
+    expect(fileContent).toMatch(/import {.*RulesEngineRunnerModule.*} from '@o3r\/rules-engine'/);
```

Comment:
```suggestion
    expect(fileContent).toMatch(/import {.*\bRulesEngineRunnerModule\b.*} from '@o3r\/rules-engine'/);
```
not sure it matters here, but it prevents matching `import { XXXRulesEngineRunnerModule }` or `import { RulesEngineRunnerModuleXXX }`